<launch>
   <include file="$(find rcvm_matrice)/launch/rcvm.launch"/>
	<node pkg="rcvm_matrice" type="indicate_movement_left.sh" name="indicate_movement"/>
</launch>

